### DISCUSSION TITLE
LLVM

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#3542] [Everyday usage] [answered]
Has anybody gotten this to work? No matter what I do, my M1 Mac keeps not recognising this as a command.

**Comment by SMillerDev** [comment, marked as answer]
There is no `llvm` command: https://llvm.org/docs/CommandGuide/index.html

**Reply by ghost** [reply]
Yes bad phrasing. It doesn't find it at all, is what I meant. I'm for example trying to install hoogle (for Haskell), this also does not work. Just wondering if I did something wrong.

**Reply by SMillerDev** [reply]
Well, without knowing any commands you tried or any error you see that's practically impossible to determine.

**Reply by carlocab** [reply]
You may wish to have a look at the `caveats` shown to you by `brew info llvm`.

**Reply by ghost** [reply]
Alright. From these comments I at least take it that it does work for other users on m1? In that case I will keep trying and the issue is most likely on my end :)

**Reply by carlocab** [reply]
Well, it depends on what you mean by "does work" or "doesn't work", I guess. Without more information, it's not possible to tell. In particular, we'd need to know, in precise detail:
1. what you did (e.g. what commands you ran, etc.)
2. what happened (i.e. what was the terminal output of the step above -- in text, no screenshots, please)
3. what you expected to happen

**Reply by ghost** [reply]
It's solved. It was a combination of known problems with specific ghc, stack and cabal versions, in addition to a self created permission issue.  I'll mark the first reply as answer.
